### ISSUE TITLE
Support `maxnodes` in `--nodes` option

### ISSUE THREAD

**Issue by natefoo** [natefoo/slurm-drmaa#4]
As per the [sbatch documentation](https://slurm.schedmd.com/sbatch.html), it is possible to request both a minimum and a maximum number of nodes with `--nodes`:

> -N, --nodes=<*minnodes*[*-maxnodes*]>
Request that a minimum of *minnodes* nodes be allocated to this job. A maximum node count may also be specified with *maxnodes*.

However, slurm-drmaa doesn't support this:

```pytb
Traceback (most recent call last):
  File "/home/ndc/drmaa-venv/bin/sbatch-drmaa", line 20, in <module>
    jobid = s.runJob(jt)
  File "/home/ndc/drmaa-venv/lib/python2.7/site-packages/drmaa/session.py", line 314, in runJob
    c(drmaa_run_job, jid, sizeof(jid), jobTemplate)
  File "/home/ndc/drmaa-venv/lib/python2.7/site-packages/drmaa/helpers.py", line 303, in c
    return f(*(args + (error_buffer, sizeof(error_buffer))))
  File "/home/ndc/drmaa-venv/lib/python2.7/site-packages/drmaa/errors.py", line 151, in error_check
    raise _ERRORS[code - 1](error_string)
drmaa.errors.InvalidArgumentException: code 4: not an number: 1-1
```

**Comment by natefoo** [comment]
Turns out [it is supported](https://github.com/natefoo/slurm-drmaa/blob/57ebc0cf7fa14f34f55f67a128ec5115e6cb60e6/slurm_drmaa/util.c#L240) but the syntax is `-N <min>=<max>`. I think this was probably a typo, even the [slurm-drmaa documentation](https://github.com/natefoo/slurm-drmaa/blob/master/README.md) says it should be `<min>-<max>`. So I think the correct thing to do here would be to support the correct syntax.

**Comment by natefoo** [comment]
The delimiter has been corrected as of 09a90b62b1dce6d741b2dba3688a747e3e564bb9

**Comment by brunogrande** [comment]
Thanks, @natefoo! I will let you know if I run into issues with the latest version once I have jobs to test it with.
